app/views/publishers/verification_dns_record.html.slim

```diff
@@ -1,8 +1,22 @@
 - content_for(:navbar_content) do
   span Step 2 of 4: #{I18n.t("publishers.new_header")}
 
+javascript:
+  function submit_for_verification () {
+    var dns_records = document.getElementById('dns_records');
+    var verification_in_progress = document.getElementById('verification_in_progress');
+    var submit = document.getElementById('begin_verification');
+
+    dns_records.style.display = 'none';
+    verification_in_progress.style.display = 'block';
+
+    setTimeout(function () {
```

Comment:
Good catch :smile: I created an issue where this can be fixed: https://github.com/brave/publishers/issues/43